src/main/java/org/jetbrains/kotlinx/lincheck/strategy/Strategy.kt

```diff
@@ -35,9 +36,27 @@ abstract class Strategy protected constructor(
     val scenario: ExecutionScenario
 ) {
     open fun needsTransformation() = false
-    open fun createTransformer(cv: ClassVisitor?): ClassVisitor {
+    open fun createTransformer(cv: ClassVisitor): ClassVisitor {
```

Comment:
It would be super-cool if we can somehow split the strategy logic itself with the JVM-related issues (such as code transformation) -- that would help a lot to make Lincheck multiplatform.